frontend/public/components/utils/_resource-log.scss

```diff
@@ -0,0 +1,16 @@
+.resource-log {
+  height: 100%;
+  width: 100%;
+}
+
+.resource-log--fullscreen {
+  .co-toolbar {
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+}
+
+.resource-log__action-divider {
```

Comment:
I think this should be part of the `.co-toolbar` styles [here](https://github.com/openshift/console/blob/20b586c34ef0bfad7fe2f8bef5243f10eb650d43/frontend/public/style/_common.scss#L227-L268)

so this would become `.co-toolbar__item--divider`